### DISCUSSION TITLE
why download the chrome driver every time

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [ultrafunkamsterdam/undetected-chromedriver#1664] [Q&A]
I have read the source code and found download chrome driver every time when start undetect chrome driver. is it need to redownload? and why?

**Comment by emanuelazfernandes** [comment]
@jiangxiaoqiang I think the intended use is to facilitate keeping a driver up to date and patch it. The intended use, that should be clearer in the documentation, is to pass the argument driver_executable_path when initializing the Chrome() class, like so:
`driver = uc.Chrome(driver_executable_path="/home/<your_username>/.local/share/undetected_chromedriver/undetected_chromedriver")`
So, I would recommend using just simple Chrome() once in a while, let the script do it's thing, and then use the above option so it doesn't download-and-patch everytime.
